app/src/main/java/com/github/polypoly/app/base/game/location/InGameLocation.kt

```diff
@@ -0,0 +1,77 @@
+package com.github.polypoly.app.base.game.location
+
+import com.github.polypoly.app.base.game.Player
+
+/**
+ * A class that represent an location in a game
+ * @property location The location of the in-game location, that permit to know the static info of
+ * the location as the name, the base price, the base tax price, etc...
+ * @property level The current level of the location that determine the price and the tax price
+ * @property owner The owner of the location, if the location is not owned, the owner is null
+ * @property bets The list of the bets made on the location by the players in the previous round
+ */
+data class InGameLocation (
+    var location: Location,
+    var level: LocalizationLevel = LocalizationLevel.LEVEL_0,
+    var owner: Player? = null,
+    val bets: List<LocationBet> = listOf()
```

Comment:
I don't think keeping the bet list as an argument of the `InGameLocation`. Maybe adding a hashmap in the Game logic itself could be a better idea, as doing a bet is common to all the players and modifying the player state is the Game role.
Tell me if you don't understand me :)

Reply:
I don't agree, it modularizes better because we don't just have a big Game class that does everything like that. The only thing I should have done though is not to put it in the constructor, I'm going to fix that right away.